src/dialogs/more-info/controls/more-info-alarm_control_panel.html

```diff
@@ -97,25 +93,27 @@
 
   stateObjChanged(newVal, oldVal) {
     if (newVal) {
-      this.codeFormat = newVal.attributes.code_format;
-      this.codeInputVisible = this.codeFormat !== null;
-      this.codeInputEnabled = (
+      const props = {
+        codeFormat: newVal.attributes.code_format,
+        armHomeButtonVisible: newVal.state === 'disarmed',
+        armAwayButtonVisible: newVal.state === 'disarmed',
+      };
+      props.codeInputEnabled = (
```

Comment:
arlo also has the state `armed`

`props.codeInputEnabled = props.disarmButtonVisible || newVal.state === 'disarmed'`